templates/2025/home/all_years.html

```diff
@@ -513,7 +513,7 @@ <h4 class="h5 g-color-black g-mb-5">Paul Everitt</h4>
                       <div class="g-mb-30">
                         <h4 class="h5 g-color-black g-mb-5">Edidiong Asikpo</h4>
                         <em class="d-block u-info-v1-5__item g-color-primary g-font-style-normal g-font-size-11 text-uppercase">Software Engineer</em>
-                        <span class="u-label u-label--sm g-bg-gray-light-v4 g-color-primary g-rounded-15"><b>Panel Session:</b> The Future of Open Source in Africa</span>                  
+                        <span class="u-label u-label--sm g-bg-gray-light-v4 g-color-primary g-rounded-15"><b>Panel Session:</b> The Future of Open Source in Uganda</span>                  
                      
```

Comment:
maybe we could remove this paragraph....seems the event schedule isnt yet drafted